.gitignore

```diff
@@ -1,23 +1,451 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
+# moban hashes
+.moban.hashes
 
-# compiled output
-/dist
-/tmp
+# Patch & backup files
+*.patch
+*.orig
+*.diff
 
-# dependencies
-/node_modules
-/bower_components
+# Extra rules from https://github.com/github/gitignore/
+# Node rules
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# TypeScript v1 declaration files
+typings/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
 
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# next.js build output
+.next
+
+# vuepress build output
+.vuepress/dist
+
+# Serverless directories
+.serverless
+# Sass rules
+.sass-cache/
+*.css.map
+*.sass.map
+*.scss.map
+
+# Emberjs rules
 # misc
-/.sass-cache
 /connect.lock
-/coverage/*
 /libpeerconnection.log
-npm-debug.log*
-yarn-error.log
 testem.log
 
 # ember-try
 .node_modules.ember-try/
 bower.json.ember-try
 package.json.ember-try
+
+
+# VirtualEnv rules
+# Virtualenv
+# http://iamzed.com/2009/05/07/a-primer-on-virtualenv/
+.Python
+[Bb]in
+[Ii]nclude
```

Comment:
`include` This seems normal in a js project.

Reply:
Is it a problem for this repo?

https://gitlab.com/coala/mobans/issues/34 is unrelated to this.
If there is a problematic rule, we override with a` !rule`

Reply:
If I add an include directory and it will be missing in git status.

Reply:
Shouldn't git ignore should only ignore files that relevant to its project?

Reply:
```
+targets:
+  - .gitignore: coala-gitignore.jj2
```
I thought it was because it is using `coala-gitignore.jj2` which causing it include python specific project ignore to this project's `.gitignore`

Reply:
This repo is configured to use the node ignore list. Nothing to do with python.
'coala' in coala-ignore filename refers to coala org, not coala tool. These are all global rules for editors,  and are all from github.com repos.

Reply:
See https://github.com/coala/gh-board/issues/50 which is about a real problem, and how to detect real problems.